packages/web3torrent/scripts/start.js

```diff
@@ -69,7 +69,6 @@ void (async () => {
   });
 
   trackerServer.stderr.on('data', data => {
-    if (process.env.DEBUG !== 'web3torrent:tracker') throw data.toString();
```

Comment:
any particular reason this error throwing logic was removed?

Reply:
The error that the tracker throws, closes the tracker... when it shouldn't. 
It's a really simple app, that I've never seen fail, so... I took it out.